src/fswAlgorithms/attGuidance/oneAxisSolarArrayPoint/_UnitTest/test_oneAxisSolarArrayPoint.py

```diff
@@ -0,0 +1,263 @@
+    rS_B = np.matmul(BN, rHat_SB_N)
+    NavAttMessageData = messaging.NavAttMsgPayload()     
+    NavAttMessageData.sigma_BN = sigma_BN
+    NavAttMessageData.vehSunPntBdy = rS_B
+    NavAttMsg = messaging.NavAttMsg().write(NavAttMessageData)
+    attReferenceCongfig.attNavInMsg.subscribeTo(NavAttMsg)
+
+    if bodyAxisInput == 0:
+        attReferenceCongfig.h1Hat_B = hHat_B
+    else:
+        # Create input bodyHeadingMsg
+        bodyHeadingData = messaging.BodyHeadingMsgPayload()
+        bodyHeadingData.rHat_XB_B = hHat_B
+        bodyHeadingMsg = messaging.BodyHeadingMsg().write(bodyHeadingData)
+        attReferenceCongfig.bodyHeadingInMsg.subscribeTo(bodyHeadingMsg)
+
+    if inertialAxisInput == 0:
+        attReferenceCongfig.hHat_N = hHat_N
+    elif inertialAxisInput == 1:
+        # Create input inertialHeadingMsg
+        inertialHeadingData = messaging.InertialHeadingMsgPayload()
+        inertialHeadingData.rHat_XN_N = hHat_N
+        inertialHeadingMsg = messaging.InertialHeadingMsg().write(inertialHeadingData)
+        attReferenceCongfig.inertialHeadingInMsg.subscribeTo(inertialHeadingMsg)
+    else:
+        # Create input translation navigation
+        transNavData = messaging.NavTransMsgPayload()
+        transNavData.r_BN_N = [0, 0, 0]
+        transNavMsg = messaging.NavTransMsg().write(transNavData)
+        attReferenceCongfig.transNavInMsg.subscribeTo(transNavMsg)
+        ephemerisData = messaging.EphemerisMsgPayload()
+        ephemerisData.r_BdyZero_N = hHat_N
+        ephemerisMsg = messaging.EphemerisMsg().write(ephemerisData)
+        attReferenceCongfig.ephemerisInMsg.subscribeTo(ephemerisMsg)
+
+
+    # Setup logging on the test module output message so that we get all the writes to it
+    dataLog = attReferenceCongfig.attRefOutMsg.recorder()
+    unitTestSim.AddModelToTask(unitTaskName, dataLog)
+
+    # Need to call the self-init and cross-init methods
+    unitTestSim.InitializeSimulation()
+
+    # Set the simulation time.
+    # NOTE: the total simulation time may be longer than this value. The
+    # simulation is stopped at the next logging event on or after the
+    # simulation end time.
+    unitTestSim.ConfigureStopTime(macros.sec2nano(1.0))
+
+    # Begin the simulation time run set above
+    unitTestSim.ExecuteSimulation()
+
+    moduleOutput = dataLog.sigma_RN
+    sigma_RN = moduleOutput[0]
+    RN = rbk.MRP2C(sigma_RN)
+    NR = RN.transpose()
+    a1Hat_N = np.matmul(NR, a1Hat_B)
+    gamma_sim = np.arcsin( abs( np.clip( np.dot(rHat_SB_N, a1Hat_N), -1, 1 ) ) )
+
+    # set the filtered output truth states
+    if alignmentPriority == 0:
+        if not unitTestSupport.isDoubleEqual(gamma_sim, gamma_true, accuracy):
+            testFailCount += 1
+            testMessages.append("FAILED: " + attReferenceWrap.ModelTag + " Module failed incidence angle for bodyAxisInput = {}, inertialAxisInput = {} and priorityFlag = {}".format(bodyAxisInput, inertialAxisInput, alignmentPriority))
```

Comment:
:small_red_triangle_down: Reformat these lines by making it multi-line. Typically we would want to limit line length in code to 120 characters. Yes we often have wide screens, but not all the time, in all environments.

Reply:
Done